packages/config/config/devices/0x031e/lzw60.json

```
@@ -0,0 +1,205 @@
+// Inovelli LZW60
+// 4-in-1 Sensor
+{
+	"manufacturer": "Inovelli",
+	"manufacturerId": "0x031e",
+	"label": "LZW60",
+	"description": "4-in-1 Sensor",
+	"devices": [
+		{
+			"productType": "0x000d",
+			"productId": "0x0001",
+			"zwaveAllianceId": 3723
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"paramInformation": {
+		"10": {
+			"label": "Low Battery Power Level Alarm",
+			"valueSize": 1,
+			"unit": "%",
+			"minValue": 10,
+			"maxValue": 50,
+			"defaultValue": 10,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"12": {
+			"label": "PIR Sensitivity",
+			"description": "Change the sensitivity of the PIR (motion) sensor; 0 = off, 1 (low) - 10 (high)",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 10,
+			"defaultValue": 10,
```

My manual for this device lists default as 8.
```suggestion
			"defaultValue": 8,
```